nodes/src/nodes/cache_llm/cache_client.py

```
@@ -0,0 +1,297 @@
+        except Exception:
+            return None
+
+    def _redis_set(self, key: str, response: dict, ttl: int) -> None:
+        """Set in Redis, silently ignoring errors."""
+        if self._redis_client is None:
+            return
+        try:
+            serialized = json.dumps(response, ensure_ascii=True)
+            self._redis_client.setex(self._key_prefix + key, ttl, serialized)
+        except Exception:
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)